docs/fetching-incidents/README.md

```diff
@@ -0,0 +1,59 @@
+# Fetching Incidents
+
+## Overview
+Demisto can pull events from 3rd party tools and convert them into actionable incidents. There are a few important parts that are necessary to keep in mind while doing so and they are outlined below.
```

Comment:
I think it will be nice if we will add screenshot of an alert coming from external product, we fetch it every minute, goes through classification &mapping => preprocessing => Incident in demisto => executing playbook

Reply:
I think we will be able to use this diagram in other places also

Reply:
sounds like a video snippet from a demo will be nice

Reply:
We can link into the tutorial video at a specific time stamp if we'd like. Alternatively, I can even do some gifs that match along with the process.